submissions/aymkin/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,276 @@
+/* NAV START */
+.container {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    width: 340px;
+    position: absolute;
+    right: 10px;
+    top: -10px;
+}
+
+.element {
+    margin: 5px;
+}
+
+.tiel {
+    background-image: url(img/tile.png);
+    width: 32px;
+    height: 32px;
+}
+
+.icon {
+    width: 32px;
+    height: 32px;
+    border-radius: 50%;
+    background-image: url(img/account.png);
+}
+
+.element a {
```

Comment:
Avoid styling by tag add custom styles on class.
For `<body>` leave but for other tags it is better to assign them a class and style them according to the class.
When a project is large and the team is working on it, it avoids the constant dropping of styles for the tag in different places. In addition, the class can be applied to different tags.

Reply:
gotcha